test/fiddle/test_pointer.rb

```diff
@@ -187,6 +187,19 @@ def test_cmp
       assert_nil(ptr <=> 10, '10 should not be comparable')
     end
 
+    def test_ref
+      ptr = Fiddle::Pointer["hello"]
+      ref = ptr.ref
+      assert_equal 0, ref.size
+      assert_nil ref.free
+      assert_equal ptr, ref.ptr
```

Comment:
Could you use parenthesis for assertions?
See also: https://bugs.ruby-lang.org/issues/20922

```suggestion
      assert_equal(0, ref.size)
      assert_nil(ref.free)
      assert_equal(ptr, ref.ptr)
```